### PR TITLE
Remove oxford commas

### PR DESCRIPTION
**Pull request by jeffu**

### CODE REVIEW

**Comment by Cbeck527** [comment]
Here we go...

:neutral_face:

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ You'll have a new folder in the directory called `_site` that has all of the com
 
 ## Deployment
 
-For deploying to S3, we use a tool called `s3_website` which looks at a file named `s3_website.yml` for configuration information. Take a look at the example file, add you S3 API keys and bucket name, and save the file as `s3_website.yml`
+For deploying to S3, we use a tool called `s3_website` which looks at a file named `s3_website.yml` for configuration information. Take a look at the example file, add you S3 API keys and bucket name and save the file as `s3_website.yml`
```
Comment (jeffu) [inline README.md]: @Cbeck527 Perhaps we can compromise and add the comma back here and not in `_config.yml`? I can see it making sense here for clarity, given the "and" in the previous clause.

Comment (Cbeck527) [inline README.md]: @jeffu hahaha I accept the compromise... FOR NOW :wink:

Comment (Cbeck527) [inline README.md]: @jeffu we should also fix the typo in this PR as well

> ... add **your** S3 API keys...

**Pull request merged**